components/general/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import { textFont, titleFont } from "@/pages/_app";
+import styles from "@/styles/Card.module.css";
+
+interface Props {
+  image: string;
+  cornerMakerLeft?: string;
+  title?: string;
+}
+
+const Card = ({ image, title, cornerMakerLeft }: Props) => {
+  return (
+    <section className={styles.card_list}>
```

Comment:
I think the class name of this component is not appropriate. If I'm not wrong, this is a single card item but you use the keyword "list" as the class name. I think it can confuse others.

Reply:
Rename to just "card"